_data/applications.yml

```diff
@@ -196,3 +196,13 @@ tukit:
   description: |
     Upgrade systems based on transactional-update from OpenSUSE with tukit in
     your browser.
+    
+sensors:
+  title: Sensors
+  package: sensors-plugin
```

Comment:
Is the Linux distribution package called `sensors-plugin` and not `cockpit-sensors` (or even `cockpit-sensors-plugin`)?

Reply:
I made the correction in the module name

Reply:
Sorry for the delay